docs/book/stacks-and-components/component-guide/image-builders/kaniko.md

```diff
@@ -139,6 +140,7 @@ zenml image-builder register <NAME> \
     --kubernetes_context=<KUBERNETES_CONTEXT> \
     --volume_mounts='[{"name": "docker-config", "mountPath": "/kaniko/.docker/"}]' \
     --volumes='[{"name": "docker-config", "configMap": {"name": "docker-config"}}]'
+    # --executor_args='["--compressed-caching=false", "--use-new-run=true"]'
```

Comment:
why has this been commented out?

Reply:
It's just there as an illustration?

Reply:
Yes exactly because the options are only needed for specific cases